package.json

```diff
@@ -32,11 +33,11 @@
     "constraint"
   ],
   "devDependencies": {
-    "apollo-server-express": "3.11.1",
+    "apollo-server-express": "^3.11.1",
+    "@apollo/server": "^4.3.0",
     "coveralls": "3.1.1",
     "express": "4.18.2",
-    "express-graphql": "0.12.0",
```

Comment:
Re-add, the module may now be deprecated but this is a breaking change to remove support here

Reply:
Problem is that for Apollo 4 I had to use graphql 16, but express-graphql requires graphql 15. This is why I deprecated `validation-rule-express-graphql` plugin and removed tests for it (as tests doesn't work as this dependency can't be here)

Reply:
Ideally there should be a way how to run some plugin's tests with different dependencies, but it would probably require breakup to more submodules? Not sure if it is worth for now.

Reply:
BTW this is only in `devDependencies`, in `peerDependencies` we still have `"graphql": ">=14.0.0"` so if someone uses this module in a project with `graphql` 15 it will work, that `validation-rule-express-graphql` plugin will work so there is not a real breaking change. But that plugin is not tested anymore so can be broken in the future, this is why I marked it as deprecated.